sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveMetadataCacheSuite.scala

```diff
@@ -126,4 +129,39 @@ class HiveMetadataCacheSuite extends QueryTest with SQLTestUtils with TestHiveSi
   for (pruningEnabled <- Seq(true, false)) {
     testCaching(pruningEnabled)
   }
+
+  test("cache TTL") {
```

Comment:
Could you make the title clearer?

Reply:
Fixed in https://github.com/apache/spark/pull/28852/commits/7d94f943496fb463f18cd445ccbde3f5225c8e7b